### ISSUE TITLE
Strong-Name Sign Nuget Package

### ISSUE THREAD

**Issue by seahckr** [FlorianHockmann/Gremlin.Net#12]
Hello! The latest nuget package 0.2.1 doesn't seem to be strong-name signed.

CSC : error CS1577: Assembly generation failed -- Referenced assembly 'Gremlin.Net' does not have a strong name

Is this an oversight?

**Comment by FlorianHockmann** [comment]
It's not really an oversight. The reason why it's not signed is simply that no one requested that until now and [Microsoft seems to advocate against the strong-name signing system nowadays](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md).

Do you need a strong-named version of Gremlin.Net?

**Comment by seahckr** [comment]
Yes please! I am trying to consume Gremlin.net from an assembly that's strong signed, so I'm not able to use it. If you can ship a strong-name signed version of Gremlin.Net to nuget, that would be awesome. Thank you @FlorianHockmann

**Comment by ashwini-ms** [comment]
@FlorianHockmann , Any update on Strong-Name Sign Nuget Package? We are blocked on this because of other dependencies. Any ETA ?

Thanks 
Ashwini

**Comment by FlorianHockmann** [comment]
Sorry for the delay. I mainly concentrated on the extended version of Gremlin.Net that is now part of Apache TinkerPop (see the README.md for more information).
But I just published version [0.2.2](https://www.nuget.org/packages/Gremlin.Net/0.2.2) which is strong-name signed.

I also created a [ticket](https://issues.apache.org/jira/browse/TINKERPOP-1785) to strong-name sign Gremlin.Net in TinkerPop
